function.k

```diff
@@ -359,10 +404,51 @@ rule <k> pcsContractPart(C:Id, function F:Id (P1:FParameters) .FunQuantifiers
 	       ...
            </Function>
 
+      rule <k> UpdateTemp(CI) => . ... </k>   
+           <contractStack> ListItem(CI:Int) ... </contractStack>
+           <contractInstance>
+     	       <ctId> CI </ctId>
+               <ctContext> Rho </ctContext>
+               <ctstore> S </ctstore>
+               <distore> Ds </distore>
+               <Memory> M </Memory>
+               <tempContext> Rho1=>Rho </tempContext>
+               <tempStore> S1=>S </tempStore>
+               <tempdistore> Ds1 => Ds </tempdistore>
+               <tempmemory> M1 => M </tempmemory>
+               ...
+           </contractInstance>  
+           <funCond> _=>true </funCond>
```

Comment:
Can we find a better name for this? 
With other things around like `loopCon` etc., it gets confusing. 

I think we should make an effort to find meaningful names in order to make our lives easier.